### PR TITLE
Aurora update v1.3.2

### PR DESCRIPTION
**Pull request by go-aurora-engine**
aurora 升级到 v1.3.2 优化了RESTFul 参数解析

### CODE REVIEW

File: go/aurora/config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 framework:
   gitee: aurora-engine/aurora
-  version: 1.0
+  version: 1.3.2
```
Comment (waghanza) [inline go/aurora/config.yaml:3]:
```suggestion
  version: 1.3
```

File: go/aurora/go.mod [hunk 2]
```diff
@@ -1,4 +1,5 @@
 module main
 
+go 1.19
```
Comment (waghanza) [inline go/aurora/go.mod:3]:
```suggestion
```

**Review by waghanza — CHANGES_REQUESTED** [review]
🎉

File: go/aurora/go.mod (hunk 2 above)
Comment (go-aurora-engine) [reply go/aurora/go.mod:3]: Do I need to delete module main?

**Review by waghanza — APPROVED** [review]

**Pull request merged**